src/TouristInfoCards.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+
+const TouristInfoCards = () => {
+  return (
+    <div className="card-container">
+      <div className="card">
+        <img
+          src="https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcS5UJXnzoIfjqkqg3higrlaxdyL_NSybHvgjWPLRw3Cgh_XESi4&usqp=CAU"
+          className="card-img-top"
+        />
+        <div className="card-body">
+          <a
+            href="https://peoplemakeglasgow.com/"
+            target="_blank"
+            className="btn btn-primary"
+            rel="noopener noreferrer"
+          >
+            Glasgow
+          </a>
+        </div>
+      </div>
+      <div className="card">
+        <img
+          src="https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcRHN5AORl_197n6KgEh-1Ag8yloH5K-ZKhNd4WH5oyx8l-uiMpI&usqp=CAU"
+          className="card-img-top"
+        />
+        <div className="card-body">
+          <a
+            href="https://www.visitmanchester.com/"
+            target="_blank"
+            className="btn btn-primary"
+            rel="noopener noreferrer"
+          >
+            Manchester
+          </a>
+        </div>
+      </div>
```

Comment:
Don't you think we could make this `<div className="card">` a component, and then, make it receive that info via props?

Another point, it is always important to try to don't repeat code. So, in this case, what if we created a data structure which we could map and return this component? Wouldn't be cool? We could avoid copy and paste this component over and over again.

Something like:
```js
const touristInformations = [{
   city: 'Glasgow',
   site: 'https://peoplemakeglasgow.com/',
   imageURL: '"https://frontline-collections.com/wp-content/uploads/2018/02/Glasgow-Debt-Collection.jpg'
}, {
   city: 'Manchester',
   site: 'https://www.visitmanchester.com/',
   imageURL: 'https://www.prolificnorth.co.uk/sites/default/files/styles/lightbox_large/public/images/news/manchester-cityscape0.jpg?itok=VjQDdTYJ'
}];
```

What do you think?

Reply:
Great advice. I have re-constructed the code as you advised. Thanks